src/components/messageFeed/MessageFeed.js

```diff
@@ -31,6 +31,12 @@ class MessageFeed extends Component {
     let messages = this.props.messages.map(message => {
       return (
         <React.Fragment key={message.id}>
+          {/* 
+          
+          ECS: Consider just passing the entire message object and not passing so many individual values
+          from the same object
+          
+          */}
```

Comment:
Eric, you wrote this code when showing me how to do the message feed by grabbing the messages without making an unnecessary get request.